### ISSUE TITLE
Azure Function Deploy from Deployment Center Stopped Working Again

### ISSUE THREAD

**Issue by evenancio** [projectkudu/kudu#2838]
Hello all

@davidebbo Not sure if you remember that https://github.com/Azure/app-service-announcements-discussions/issues/64#issuecomment-414829772

But the same problem is happening again unfortunately.

Since this morning as far as we noticed that, our functions deploys integrated with Deployment Options stopped working again.

Let me describe my problem again, just in case someone else can help me:

Although our app functions are pinned to 2.0.11961-alpha, the Deployment Center Logs are saying "Your function app is targeting V1, but Azure host has function version V2, \r\nplease change the version using the portal or update your 'FUNCTIONS_EXTENSION_VERSION' appsetting and retry" during the "Preparing deployment for commit id..." phase.

We already tried to disconnect and redo the integration with VSTS, but no luck so far. Do someone know what is going on? I'm freaking out because of that.

In addition, we have some App Functions integrated with VSTS. By the way, if we deploy it from VS Code Extension, everything works as expected.

@davidebbo helped me with that a couple of weeks ago, but we are having the same problems again now.

Thanks

**Comment by davidebbo** [comment]
Have you tried restarting the app to make sure you have the latest Kudu? WHat Kudu version do you see when going to the root of scm site?

**Comment by evenancio** [comment]
Hi @davidebbo . Sorry for bothering you again with this subject.

Yes, I've tried to restart server and redeploy it a few times, but no luck so far.

I also double checked my Application Settings, and there is no KUDU_* variables anymore.

According to the *.scm.azurewebsites.net/Env.cshtml it looks like we are running the following Kudu Version: 77.10823.3533 (D:\Program Files (x86)\SiteExtensions\Kudu\77.10823.3533\)

Build 77.10823.3533.0 (e4e2ef0302)
Azure App Service 76.0.8598.33 (rd_websites_stable.180727-1620)

**Comment by evenancio** [comment]
Sorry guys, but have you had a chance to take a look at this problem?

I just restarted my function app and tried to redeploy before writing this message, but it is not working yet.

**Comment by davidebbo** [comment]
Can you please share the app name, and time of deployment?

**Comment by evenancio** [comment]
Sure

For example: CTRwow-Assets-Microservice

I tried to deploy it just a few minutes ago, 08:10pm UTC

Thanks in advance. I really appreciate your time trying to help us on that.

**Comment by evenancio** [comment]
Hi David

Firstly, it was the cause of my problem, so after updating my csproj, everything back to normal again. Thank you for that. 

That's really weird. Our "older" functions apps have their csproj set like that (<AzureFunctionsVersion/>). The new ones are correct though.

I took a look at Commits History and I realized these older functions apps have this wrong configuration since the initial commit, so I suppose we were using an old version of Azure Functions Core Tools, which generated our project like that - just my guess, I don't know to be honest.

Anyway, it's working now. Hopefully we won't have this problem again!

Thank you very much!
